ksql-engine/src/test/java/io/confluent/ksql/function/UdfArgCoercerTest.java

```diff
@@ -28,14 +42,14 @@ public void testCoerceNumbers() {
 
     // Then:
     for (int i = 0; i < args.length; i++) {
-      assertThat(UdfTemplate.coerce(args, int.class, i), equalTo(1));
-      assertThat(UdfTemplate.coerce(args, Integer.class, i), equalTo(1));
+      assertThat(UdfArgCoercer.coerceUdfArgs(args, int.class, i), Matchers.equalTo(1));
```

Comment:
nit: can we add a static import for `equalTo` please

```suggestion
      assertThat(UdfArgCoercer.coerceUdfArgs(args, int.class, i), equalTo(1));
```